components/03-organisms/menu/primary-nav/_yds-primary-nav.scss

```diff
@@ -184,12 +214,38 @@
   &--level-1 {
     font: var(--font-style-nav-primary-1);
     color: var(--color-heading);
-    padding: var(--size-spacing-2) var(--size-spacing-5);
     margin-bottom: var(--size-spacing-3);
 
+    [data-menu-variation='basic'] & {
+      padding: var(--size-spacing-2) var(--size-spacing-6) var(--size-spacing-4);
+      position: relative;
+
+      // basic menu items have a bottom border
+      &::after {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        left: var(--size-spacing-5);
+        right: var(--size-spacing-5);
+        height: var(--border-thickness-1);
+        background-color: var(--color-gray-200);
+      }
+    }
+
+    [data-menu-variation='mega'] & {
+      padding: var(--size-spacing-2) var(--size-spacing-5);
+    }
+
     .primary-nav__item--level-1:first-child & {
       padding-top: 0;
     }
+
+    // remove last child's bottom border
+    [data-menu-variation='basic'] .primary-nav__item--level-1:last-child & {
+      &::after {
+        content: unset;
+      }
+    }
```

Comment:
This works fine, but I would use a `:not()` up above to not ever add this in the first place.

So:
```
      // basic menu items have a bottom border
      &:not(:last-child)::after {
        content: '';
        position: absolute;
        bottom: 0;
        left: var(--size-spacing-5);
        right: var(--size-spacing-5);
        height: var(--border-thickness-1);
        background-color: var(--color-gray-200);
      }
    }
```

Then this chunk of code isn't needed.